x-pack/test/functional/apps/ml/data_frame_analytics/classification_creation.ts

```diff
@@ -99,38 +83,63 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.dataFrameAnalyticsCreation.setTrainingPercent(testData.trainingPercent);
         });
 
+        it('continues to the additional options step', async () => {
+          await ml.dataFrameAnalyticsCreation.continueToNextStep(
+            'mlAnalyticsCreateJobWizardAdvancedStep'
```

Comment:
I'd rather have this `data-test-subj` hidden in a re-usable service method, e.g. `ml.dataFrameAnalyticsCreation.continueToAdditionalOptionsStep()`

(this is for all `continueToNextStep` calls in the test files, I will not repeat this comment)

For the AD job wizard we have something similar, see
https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/ml/job_wizard_common.ts#L60

Reply:
Updated in 0406a647b262b1c0a943f03e3056141005938aff